### PR TITLE
Remove Twig 2 support

### PR DESCRIPTION
**Pull request by loic425**
| Q               | A
| --------------- | -----
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | 
| License         | MIT

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -83,14 +83,15 @@
...
+        "twig/twig": "^3.0",
         "vimeo/psalm": "^5.20",
         "willdurand/hateoas-bundle": "^2.0"
     },
     "conflict": {
         "friendsofsymfony/rest-bundle": "<3.0",
         "jms/serializer-bundle": "<3.5",
-        "willdurand/hateoas-bundle": "<2.0 || ^2.6"
+        "willdurand/hateoas-bundle": "<2.0 || ^2.6",
+        "twig/twig": "<3.0"
```
Comment (GSadee) [inline composer.json:94]: As we have bumped up the dependency, I'm not sure if it is necessary to add this package additionally to the conflicts (and the about note to the `CONFLICTS` file), do you see any benefit of that? 🤔

Comment (loic425) [reply composer.json:94]: cause it's on require-dev, it needs a conflict. We often forgot about those conflicts when there are on "optional/dev" requirements.
However, I do not know if adding note in the conflict make sense, cause it's just a bumped dependency.

Comment (loic425) [reply composer.json:94]: I've removed CONFLICTS file part.

**Review by GSadee — APPROVED** [review]

**Comment by GSadee** [comment]
Thank you, Loïc! :tada:

**Pull request merged**